pkgs/applications/kde/kalgebra.nix

```diff
@@ -0,0 +1,46 @@
+{ mkDerivation
+, lib
+, extra-cmake-modules
+, kdoctools
+, analitza
+, ki18n
+, kinit
+, kirigami2
+, kconfigwidgets
+, kwidgetsaddons
+, kio
+, kxmlgui
+, qtwebengine
+, plasma-framework
+}:
+
+mkDerivation {
+  pname = "kalgebra";
+
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [
+    extra-cmake-modules
+    kdoctools
+  ];
+
+  buildInputs = [
+    ki18n
+    analitza
+    kinit
+    kirigami2
+    kconfigwidgets
+    kwidgetsaddons
+    kio
+    kxmlgui
+    qtwebengine
+    plasma-framework
+  ];
+
+  meta = {
+    homepage = "https://apps.kde.org/kalgebra/";
+    description = "Graphing calculator";
+    license = with lib.licenses; [ gpl2 ];
+    maintainers = with lib.maintainers; [ ninjafb ];
```

Comment:
The `mainProgram` attribute is missing.

Reply:
Hi, glancing at the other .nix files in the KDE folder (with a grep command) they don't appear to have a `mainProgram` attribute either so it seems like this would break the established convention. I would also like to thank you for you very punctual reply and help :)

Reply:
Fair enough, then ! Thanks for checking it out! Let's move on :)